ev3dev2/stopwatch.py

```diff
@@ -5,7 +5,7 @@
 try:
     import datetime as dt
     micropython = False
-except ImportError:
```

Comment:
We sure we want to do that?

Reply:
yeah it is a bit odd but on my laptop (ubuntu) it raises `ImportError` but on ev3dev it raised something else (`AttributeError` maybe) so to play it safe I just caught all Exceptions.